pynestml/frontend/pynestml_frontend.py

```diff
@@ -108,31 +155,35 @@ def to_nest(input_path: Union[str, Sequence[str]], target_path=None, logging_lev
+    r"""Generate and build code for NEST Simulator.
 
     Parameters
     ----------
-    target_path
-        Path to the target directory, which should contain the generated code artifacts (target platform code and CMake configuration file).
-    nest_path
-        Path to the NEST installation, which should point to the main directory where NEST is installed. This folder contains the ``bin``, ``lib(64)``, ``include``, and ``share`` folders of the NEST install. The ``bin`` folder should contain the ``nest-config`` script, which is accessed by NESTML to perform the installation. This path is the same as that passed through the ``-Dwith-nest`` argument of the CMake command before building the generated NEST module. The suffix ``bin/nest-config`` will be automatically appended to ``nest_path``.
-
+    input_path : str **or** Sequence[str]
+        Path to the NESTML file(s) or to folder(s) containing NESTML files to convert to NEST code.
+    target_path : str, optional (default: append "target" to `input_path`)
+        Path to the generated C++ code and install files.
+    logging_level : str, optional (default: "ERROR")
+        Sets which level of information should be displayed duing code generation (among "ERROR", "WARNING", "INFO", or "NO").
+    module_name : str, optional (default: "nestmlmodule")
+        Name of the module, which will be used to import the model in NEST via `nest.Install(module_name)`.
+    store_log : bool, optional (default: False)
+        Whether the log should be saved to file.
+    suffix : str, optional (default: "")
+        Suffix which will be appended to the model"s name (internal use to avoid naming conflicts with existing NEST models).
     install_path
         Path to the directory where the generated NEST extension module will be installed into. If the parameter is not specified, the module will be installed into the NEST Simulator installation directory, as reported by nest-config.
-
-    Raises
-    ------
-    GeneratedCodeBuildException
-        If any kind of failure occurs during cmake configuration, build, or install.
-    '''
-    if install_path is not None:
-        nest_installer(target_path, nest_path, install_path)
-        # add the install_path to the python library
-        add_libraries_to_sli(install_path)
-    else:
-        nest_installer(target_path, nest_path)
+    dev : bool, optional (default: False)
+        Enable development mode: code generation is attempted even for models that contain errors, and extra information is rendered in the generated code.
+    codegen_opts : Optional[Mapping[str, Any]]
+        A dictionary containing additional options for the target code generator.
+    """
+    generate_target(input_path, target_path, target_platform="NEST", logging_level=logging_level,
+                    module_name=module_name, store_log=store_log, suffix=suffix, install_path=install_path,
+                    dev=dev, codegen_opts=codegen_opts)
```

Comment:
Isn't the call to `add_libraries_to_sli` missing here?

Reply:
It's in the NESTBuilder class.